apps/dashboard/src/components/workflow-editor/configure-workflow.tsx

```diff
@@ -0,0 +1,137 @@
+import { useFormContext } from 'react-hook-form';
+import { RouteFill } from '../icons';
+import { FormControl, FormField, FormItem, FormLabel, FormMessage } from '../primitives/form';
+import { Input, InputField } from '../primitives/input';
+import { Switch } from '../primitives/switch';
+import { Textarea } from '../primitives/textarea';
+import { TagInput } from '../primitives/tag-input';
+import { Separator } from '../primitives/separator';
+import { formSchema } from './schema';
+import * as z from 'zod';
+import { RiInformation2Line } from 'react-icons/ri';
+
+import { TooltipProvider, Tooltip, TooltipTrigger, TooltipContent } from '../primitives/tooltip';
+import { CopyButton } from '../primitives/copy-button';
+
+export function ConfigureWorkflow() {
+  const { control } = useFormContext<z.infer<typeof formSchema>>();
+  return (
+    <aside className="text-foreground-950 flex h-full w-[300px] max-w-[350px] flex-col border-l pb-5 pt-3.5 [&_input]:text-neutral-600 [&_label]:text-xs [&_label]:font-medium [&_textarea]:text-neutral-600">
+      <div className="flex items-center gap-2.5 px-3 pb-3.5 text-sm font-medium">
+        <RouteFill />
+        <span>Configure workflow</span>
+      </div>
+      <Separator />
+      <FormField
+        control={control}
+        name="active"
+        render={({ field }) => (
+          <FormItem className="flex items-center justify-between gap-2.5 space-y-0 px-3 py-2">
+            <div className="flex items-center gap-4">
+              <div
+                className="bg-success-alpha-600 data-[active=false]:shadow-neutral-alpha-100 ml-2 h-1.5 w-1.5 rounded-full [--pulse-color:var(--success)] data-[active=true]:animate-[pulse-shadow_1s_ease-in-out_infinite] data-[active=false]:bg-neutral-300 data-[active=false]:shadow-[0_0px_0px_5px_var(--neutral-alpha-200),0_0px_0px_9px_var(--neutral-alpha-100)]"
+                data-active={field.value}
+              />
+              <FormLabel>Active Workflow</FormLabel>
+            </div>
+            <FormControl>
+              <Switch checked={field.value} onCheckedChange={field.onChange} />
```

Comment:
the toggle size looks too big from what we have in the designs
![Screenshot 2024-10-22 at 00 29 28](https://github.com/user-attachments/assets/a0f99206-b78d-438b-a01b-2387fda1758a)